pkg/diagnose/diagnose.go

```diff
@@ -109,82 +87,101 @@ func (c *Collector) CollectCR(list client.ObjectList) {
 	}
 }
 
-// collect output of the "describe pod"
-func (c *Collector) collectPodDescribe(pod *corev1.Pod) {
-	cmd := exec.Command("kubectl", "describe", "pod", "-n", pod.Namespace, pod.Name)
+// CollectCRs collects the content of multiple CR types
+func (c *Collector) CollectCRs() {
+	c.CollectCR(&nbv1.BackingStoreList{
+		TypeMeta: metav1.TypeMeta{Kind: "BackingStoreList"},
+	})
+
+	c.CollectCR(&nbv1.BucketClassList{
+		TypeMeta: metav1.TypeMeta{Kind: "BucketClassList"},
+	})
+
+	c.CollectCR(&nbv1.NooBaaList{
+		TypeMeta: metav1.TypeMeta{Kind: "NooBaaList"},
+	})
+}
+
+// CollectDescribe collects output of the "describe pod" of a single pod
+func (c *Collector) CollectDescribe(Entity string, Name string) {
+	cmd := exec.Command("kubectl", "describe", Entity, "-n", options.Namespace, Name)
 	// handle custom path for kubeconfig file,
 	// see --kubeconfig cli options
 	if len(c.kubeconfig) > 0 {
 		cmd.Env = append(cmd.Env, "KUBECONFIG=" + c.kubeconfig)
 	}
 
 	// open the out file for writing
-	fileName := c.folderName + "/" + pod.Name + "-describe.txt"
+	fileName := c.folderName + "/" + Name + "-describe.txt"
```

Comment:
We should probably add to the filename the kind: 
```suggestion
	fileName := c.folderName + "/" + Name + Kind + "-describe.txt"
```

Reply:
Done